blaze-server/src/main/contraband/BlazeServerConfig.contra

```diff
@@ -0,0 +1,39 @@
+package org.http4s.server.blaze
+@target(Scala)
+
+type BlazeServerConfig {
+  socketAddress: java.net.InetSocketAddress!
+  serviceExecutor: java.util.concurrent.ExecutorService
+  idleTimeout: scala.concurrent.duration.Duration!
+  serverChannelGroupConfig: ServerChannelGroupConfig!
+  webSocketsEnabled: Boolean!
+  sslContext: javax.net.ssl.SSLContext
```

Comment:
`SSLContext` is more flexible than `TlsConfig`, but perhaps more hostile.  Maybe the dual methods we have in 0.15 weren't as bad as I initially thought.